### DISCUSSION TITLE
如何用Sublime Text 编译论文模板？

### DISCUSSION THREAD

**Discussion by wangjqang** [sjtug/SJTUThesis#1001] [提问]
请问有用过Sublime Text 编译过论文模板的同学嘛？我这这里显示没法编译，报错缺少LaTeX Error: File `sjtuthesis.cls' not found，我把 sjtuthesis.cls加入到文件夹中又显示编译错误：BLaTeX Error: File `sjtu-math-font-termes.def' not found. 有没有了解的同学可以给个建议，怎么解决呢？

**Comment by note286** [comment]
此问题与Sublime Text无关。

**Comment by wangjqang** [comment]
可以具体说一下，是哪里的问题嘛？或则如果用Sublime Text编译找个模板呢？谢谢

从 Windows 版邮件发送

发件人: Chloe
发送时间: 2024年1月5日 15:36
收件人: sjtug/SJTUThesis
抄送: wangjqang; Author
主题: Re: [sjtug/SJTUThesis] 如何用Sublime Text 编译论文模板？ (Discussion #1001)

此问题与Sublime Text无关。

**Comment by note286** [comment]
Sublime Text只是编辑器，你用记事本也没事。

你尝试把`https://github.com/sjtug/SJTUThesis/tree/master/texmf/tex/latex/sjtutex`路径下的文件和主文件放同一目录。
